pynicotine/userlist.py

```diff
@@ -24,48 +26,151 @@ def __init__(self, core, config, queue, ui_callback):
         self.core = core
         self.config = config
         self.queue = queue
+        self.users = {}
```

Comment:
`users` -> `buddies`

In other parts of the application, users might mean any connected user partaking in transfers, search requests and chats etc.